docs/source/peer_ledger_snapshot.md

```diff
@@ -0,0 +1,153 @@
+```
+
+In this example, the ledger height is `970`.
+
+A snapshot request can be submitted by issuing a command similar to:
+
+```
+peer snapshot submitrequest -C <name of channel> -b <ledger height where snapshot will be taken> --peerAddress <address of peer> --tlsRootCertFile <path to root certificate of the TLS CA>
+```
+
+For example:
+
+```
+peer snapshot submitrequest -C testchannel -b 1000 --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+**If you give a ledger height of `0`, the snapshot is taken immediately. This is useful for cases when an organization is interested in backing up its own ledger data and does not intend to share the data with another organization. Do not take these "immediate" snapshots in cases when snapshots will be evaluated from multiple peers, as it increases the likelihood that the snapshots will be taken at different ledger heights.**
+
+If the request is successful, you will see a `Snapshot request submitted successfully` message.
+
+You can list the pending snapshots by issuing a command similar to:
+
+```
+peer snapshot listpending -C testchannel --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+You will see a response similar to:
+
+```
+Successfully got pending snapshot requests [970]
+```
+
+When a snapshot has been generated for a particular block height, the pending request for that block height will no longer appear in the list. You can also verify that a snapshot has been create successfully by looking at the peer logs. You will see a message similar to `Generated snapshot block-<block number>`.
+
+To delete a snapshot request, simply exchange `submitrequest` with `cancelrequest`. For example:
+
+```
+peer snapshot cancelrequest -C testchannel -b 1000 --peerAddress 127.0.0.1:22509 --tlsRootCertFile tls/cert.pem
+```
+
+If you submit the `listpending` command again, the snapshot should no longer appear.
+
+### Contents of a snapshot
+
+Snapshots are stored on a peer in a snapshot folder that is further organized by channel name and the ledger height when the snapshot was taken: `{snapshotsFolder}/{channelName}/{blockHeight}`.
+
+As mentioned above, the snapshot contains four folders:
+
+* **Public state**
+  * This includes the latest state of all of the assets on the channel (for example, the current owner of a car).
+* **Private data hashes**
+  * Recall from our documentation on private data that while the actual transaction data of a private data transaction is not stored on the public ledger of the channel, hashes of the data are stored. This allows the private data to be verified against the hash, for example by an organization that is added to a private data collection.
+* **Transactions IDs**
+  * Starting with Fabric v2.0, entries in the block store index take the form of `TxID_BlkNum_TxNum`. This index can be filtered for the the transactions IDs that are added after the snapshot height by evaluating block number (`BlkNum`) portion of the index entry.
```

Comment:
This consists of transaction ID that have been used till the last block in the snapshot

Reply:
I guess that you missed tackling this in your latest commit. The current description under `Transactions IDs` is the implementation details that does not make sense here. So, like previous comment, if you intend to just keep the heading, this whole text can be removed and if you want to have one liner, then it can be replaced by "This consists of transaction ID that have been used till the last block in the snapshot"